libraries/wasm-jit/Source/Platform/POSIX.cpp

```diff
@@ -101,12 +107,16 @@ namespace Platform
 
 	bool describeInstructionPointer(Uptr ip,std::string& outDescription)
 	{
-		#ifdef __linux__
+		#if defined __linux__ || defined __FreeBSD__
 			// Look up static symbol information for the address.
 			Dl_info symbolInfo;
 			if(dladdr((void*)ip,&symbolInfo) && symbolInfo.dli_sname)
 			{
-				outDescription = symbolInfo.dli_sname;
+				#ifdef __linux__
+					outDescription = symbolInfo.dli_sname;
+				#else
+					outDescription.append(symbolInfo.dli_sname);
```

Comment:
Can you explain this one further? Why is .append() needed instead of = operator?

Reply:
Ah, I just tested it again and you're right, it's not needed. This is the error I was getting:

/home/ec2-user/git/eos/libraries/wasm-jit/Source/Platform/POSIX.cpp:114:32: error: no viable overloaded '='
                outDescription = symbolInfo.dli_sname;
                ~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~
/home/ec2-user/git/eos/libraries/wasm-jit/Source/Platform/POSIX.cpp:114:32: note: type 'std::string' (aka 'basic_string<char, char_traits<char>, allocator<char> >') is incomplete
                outDescription = symbolInfo.dli_sname;
                ~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~
1 error generated.

However, without the .append() use and only including <iostream> it builds.

Reply:
Can you try #include <string> instead of iostream? My guess is that would be the appropriate one to include.